src/perf-tests/README.md

```diff
@@ -0,0 +1,111 @@
+# Overview
```

Comment:
Could you add a table to give a quick overview of what each automation piece you did and its purpose?

There's a bunch of shell scripts, python scripts, scala, etc. that may take a while to grasp. A table can help give a quick overview of what component they're for, and what they are for

Reply:
done